internal/cloud/state.go

```diff
@@ -577,7 +584,13 @@ func clamp(val, min, max int64) int64 {
 }
 
 func (s *State) readSnapshotIntervalHeader(status int, header http.Header) {
-	intervalStr := header.Get("x-terraform-snapshot-interval")
+	contentType := header.Get("Content-Type")
```

Comment:
Consider adding the comment `// Only proceed if this came from tfe.v2 API` above this

Reply:
57a2345b(cr comment)